EventsExpress/ClientApp/src/containers/event-item-view.js

```diff
@@ -3,12 +3,14 @@ import { connect } from 'react-redux';
 import EventItemView from '../components/event/event-item-view';
 import Spinner from '../components/spinner';
 import get_event from '../actions/event-item-view';
+import { get_inventories_by_event_id }  from '../actions/inventory-list';
 import { join, leave, resetEvent, cancel_event, approveUser } from '../actions/event-item-view';
 
 class EventItemViewWrapper extends Component{
     componentWillMount(){    
         const { id } = this.props.match.params;
         this.props.get_event(id);
+        this.props.get_inventories_by_event_id(id);
```

Comment:
Why do you still load these invetories here? I thought you are doing that in the component already?